### PR TITLE
#164198456 feature to view and edit profile (without test):

### PR DESCRIPTION
**Pull request by danprocoder**
#### What does this PR do?
Adds feature to view and edit profile.

#### Description of Task to be completed?
1. Create profile page.
2. Consume endpoints for profile page.
3. Create reusable components fro article item, user item and error page.
4. Make profile page responsive.

#### How should this be manually tested?
1. Clone this branch.
2. Run `cd artemis-ah-frontend`
3. Run `npm install` to install the dependencies.
4. Run `npm test` to run the tests.

#### What are the relevant pivotal tracker stories?
[#164198456](https://pivotaltracker.com/story/show/164198456)

#### Screenshots (if appropriate)
Screenshot 2019-04-12 at 4 48 53 PM
Screenshot 2019-04-12 at 4 49 11 PM

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,8 +5,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
   <title>Authors Haven</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (Chrismarcel) [inline public/index.html:8]: Hey Daniel, why did you remove the viewport tag?

Comment (danprocoder) [reply public/index.html:8]: @Chrismarcel it is already in line 5. No need to include it second time.

Comment (Chrismarcel) [reply public/index.html:8]: Alright @danprocoder. Thanks for the clarity

**Review by andela-gike — APPROVED** [review]
